.circleci/config.yml

```diff
@@ -19,8 +22,8 @@ jobs:
       - run:
           command: |
             ./run ./configure.py --compiler << parameters.compiler >> --c++-standard << parameters.standard >>
```

Comment:
If doing `./configure.py <current stuff> --mode << parameters.mode >>` things get a bit faster
(can be made as a followup)

Reply:
make sense, you didn't give me the 15sec to add it :)